pages/features/list.md

```diff
@@ -9,7 +9,7 @@ General and Miscellaneous features.
 ### Mod Checker
 
 When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of verified mods. If you believe a mod is being falsely flagged, please report it in the PSS discord (``/pssdiscord``)
-<img width="750" alt="18b5977a-c6b6-4f83-80bf-61b6a1297946" src="https://github.com/FlagHater/partly-sane-skies-FlagHater/assets/106500506/24d3ee14-bc5e-42e7-8604-c2811216b457">
+![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/601f2344-ed39-47c9-bd28-e6da2c09a639)
```

Comment:
Hello Hi thats not /images/

Reply:
Don't know how to do that

Reply:
You paste that file in there

Reply:
You have to have a cloned repo on your computer. You can't use the GitHub online editor